spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+FactoryGirl.define do
+  factory :user do
+    email { 'email@email.com' }
```

Comment:
As a follow up, we're not currently running any ActiveRecord validations so in principle it doesn't matter that this isn't a sequence right now, but we'll need to update the factory when we do add those in. Alternatively, authentication would be broken if we didn't have those validations (login with e-mail is difficult if multiple accounts have the same e-mail address) so maybe those do fall under this PR, and we should both test and implement those validations on the model level, as well as update the factory. What do you think?